config.coffee

```diff
@@ -92,11 +92,7 @@ module.exports =
   EDITORIAL_CTA_BANNER_IMG: 'http://files.artsy.net/images/iphone_email.png'
   ARTSY_PARTNER_UPDATES_CHANNEL: '5762d454b5989e6f98f7799a'
   TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/gallery-insights$|^\/artsy-partner-updates$'
-  MARKETING_SIGNUP_MODAL_HEADER: 'Sign up'
-  MARKETING_SIGNUP_MODAL_COPY: 'For early access'
-  MARKETING_SIGNUP_MODAL_IMG: 'http://placekitten.com/700/1000'
-  MARKETING_SIGNUP_MODAL_SLUG: 'miami'
-  MARKETING_SIGNUP_MODAL_PHOTO_CREDIT: 'Photo by Artsy'
+  MARKETING_SIGNUP_MODALS: '[{"slug":"ca1","copy":"An art collection for every budget","image":"http://files.artsy.net/images/modal-interior-a-credited.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca2","copy":"Buy art from the best galleries and auction houses","image":"http://files.artsy.net/images/modal-interior-a-credited.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca3","copy":"An art collection for every budget","image":"http://files.artsy.net/images/modal-interior-b-credited.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca4","copy":"Buy art from the best galleries and auction houses","image":"http://files.artsy.net/images/modal-interior-b-credited.jpg","photoCredit":"Photo by Emily Johnson"}]'
```

Comment:
Nope! It's actually better to set it now, so you don't have to remember when the code gets deployed. So 👍  go for it!